tests/test_cli.py

```diff
@@ -224,10 +224,17 @@ def test_convert(run_cli, tmpdir):
 
 @requires("click")
 def test_convert_pipe():
-    cmd = """echo /prep7 | pymapdl convert """
+    cmd = ["echo", "/prep7"]
+    cmd2 = ["pymapdl", "convert"]
 
-    out = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
-    stdout = out.stdout.read().decode()
+    process_echo = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    process_pymapdl = subprocess.Popen(
+        cmd2, stdin=process_echo.stdout, stdout=subprocess.PIPE
+    )
```

Comment:
Glad that the piping worked =)

Reply:
Yes, thank you a lot!!

Reply:
Damn... that is a pretty convoluted way to run a pipe command. Nice to know!